frontend/src/DmarcReportPage.js

```diff
@@ -211,14 +211,14 @@ export default function DmarcReportPage({ summaryListResponsiveWidth }) {
       ],
       moderate: [
         {
-          name: 'passSpfOnly',
-          displayName: t`Pass Only SPF`,
+          name: 'failDkim',
+          displayName: t`Fail DKIM`,
         },
       ],
       moderateAlt: [
         {
-          name: 'passDkimOnly',
-          displayName: t`Pass Only DKIM`,
+          name: 'failSpf',
+          displayName: t`Fail SPF`,
```

Comment:
These fields need their 'name' properties to stay 'passSpfOnly' and 'passDkimOnly'. These names come from the API. The display names are good though!